api/v1beta1/azureclustertemplate_default_test.go

```diff
@@ -1169,93 +1169,6 @@ func TestNodeOutboundLBClassDefaults(t *testing.T) {
 	}
 }
 
-func TestControlPlaneOutboundLBTemplateDefaults(t *testing.T) {
```

Comment:
This deleted test tested the unused function from above (https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/5603/files#r2072129130)